.invenio

```diff
@@ -5,13 +5,13 @@ logfile = /logs/invenio-cli.log
 [cookiecutter]
 project_name = Zenodo
 project_shortname = zenodo
-project_site = zenodo.org
+project_site = zenodo-rdm.web.cern.ch
 github_repo = zenodo/zenodo-rdm
 description = Invenio RDM Zenodo Instance
 author_name = CERN
 author_email = info@zenodo.org
 year = 2020
-python_version = 3.6
+python_version = 3.8
```

Comment:
I'm getting some PTSD here, but would it be ok to use 3.9? I can imagine by the time we migrate, it would be best to be on the latest supported Python version

Reply:
I agree, but atm not many people have tested/used it with 3.9 and there have been reports in discord about issues... Im just being safe for the two weeks we got 👼 , thats why I bumped only to 3.8